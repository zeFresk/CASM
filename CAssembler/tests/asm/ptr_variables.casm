.at 10
a: .word 001
ptr_a: .word a

.start 30
sub a
jaz ptr_a
