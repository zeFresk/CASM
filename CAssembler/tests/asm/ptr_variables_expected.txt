30
10: 001
11: 010
30: 710
31: 811
